options/index.html

```diff
@@ -30,6 +30,15 @@
 						<label for="notifs-offset">minute(s) before period end</label>
 					</span>
 				</div>
+				<div class="option">
+					<fieldset>
+						<legend>Server</legend>
+						<input type="radio" name="instance" id="instance:main" value="main" required>
```

Comment:
I think a better label for this could be Schedule (since most users don't know/care that it's a server they're switching, just which schedule is displayed).

Reply:
> I think a better label for this could be Schedule (since most users don't know/care that it's a server they're switching, just which schedule is displayed).

Done.
I still think we should change the names of the options, but I'm not sure what we should call them. Do you have any suggestions?
Here's what it's like now btw:
![image](https://user-images.githubusercontent.com/37093293/221423462-df01a32c-8c06-4ef8-8745-a3a9cb582dbb.png)

Reply:
I think that Main Campus and Day School could work as labels.

Reply:
Done.
Looks like this now:
![image](https://user-images.githubusercontent.com/37093293/221424371-b0a8bab6-282c-499c-b691-12fd8d58a298.png)

Reply:
I'll leave this open until we get feedback from Mr. Chan